src/modules/local_position_estimator/BlockLocalPositionEstimator.hpp

```diff
@@ -373,6 +394,8 @@ class BlockLocalPositionEstimator : public control::SuperBlock
 	uint8_t _sensorFault;
 	uint8_t _estimatorInitialized;
 
+	bool _targetFault; // TODO add to _sensorFault
```

Comment:
To do this, we'd have to change `_sensorTimeout` and `_sensorFault` from `uint8_t` to `uint16_t`. @jgoppert which do you prefer??

Reply:
@ndepal I don't see an issue on that. I can PR together with the above change.

Reply:
https://github.com/PX4/Firmware/pull/8226